Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocDateTimeValue.css

```diff
@@ -1,15 +1,26 @@
 /* Style applied to the BocDateTimeValue. */
 
+.bocDateTimeValue,
+.bocDateValue
+{
+  min-height: var(--remotion-themed-min-height);
+}
+
+.bocDateTimeValue input,
+.bocDateValue input
+{
+  box-sizing: border-box;
+}
+
```

Comment:
please add the span prefix. and then you will want to agggregate the classes a bit. there's multiple declarations..